### PR TITLE
Add crates.io metadata

### PR DESCRIPTION
**Pull request by iradicek**

### CODE REVIEW

File: leptos_async_signal/Cargo.toml [hunk 1]
```diff
@@ -1,10 +1,16 @@
 [package]
 name = "leptos_async_signal"
+authors = ["Demiurg <contact@demiurg.hr>"]
```
Comment (adundovi) [inline leptos_async_signal/Cargo.toml]: The e-mail address should be <developers@demiurg.hr>.

File: leptos_async_signal/Cargo.toml [hunk 2]
```diff
@@ -1,10 +1,16 @@
...
+license = "MIT"
+repository = "https://github.com/demiurg-dev/leptos_async_signal"
+description = "Async signal for leptos Rust web framework."
+categories = ["web-programming", "asynchronous"]
+keywords = ["web", "reactive", "ssr", "async"]
```
Comment (adundovi) [inline leptos_async_signal/Cargo.toml]: "leptos" as a keyword?

**Review by adundovi — CHANGES_REQUESTED** [review]

File: leptos_async_signal/Cargo.toml (hunk 1 above)
Comment (iradicek) [reply leptos_async_signal/Cargo.toml]: Done, thanks!

File: leptos_async_signal/Cargo.toml (hunk 2 above)
Comment (iradicek) [reply leptos_async_signal/Cargo.toml]: Added

**Review by adundovi — APPROVED** [review]

**Pull request merged**